tools/wpt/browser.py

```diff
@@ -51,6 +51,22 @@ def get_ext(filename):
     return ext
 
 
+def get_filename(resp, default=None):
```

Comment:
I might have given this a slightly less generic name like `get_download_filename`

Reply:
yeah, that seems reasonable